crates/uv-python/src/discovery.rs

```diff
@@ -133,6 +133,12 @@ pub enum EnvironmentPreference {
     Any,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Default)]
+pub(crate) struct DiscoveryPreferences {
```

Comment:
In a later pull request, I'll consolidate methods to take this type instead of the separate preferences.